R/pkg/R/functions.R

```diff
@@ -3324,7 +3325,8 @@ setMethod("percent_rank",
 #' The difference between rank and denseRank is that denseRank leaves no gaps in ranking
 #' sequence when there are ties. That is, if you were ranking a competition using denseRank
```

Comment:
The above two lines also have `denseRank`. To be consistent, we also should rename it to `dense_rank`